.github/actions/ci-results/action.yml

```diff
@@ -1,16 +1,11 @@
 name: 'Capture test results'
+inputs:
+  include-reports:
```

Comment:
Is this input used here?

Reply:
Urgh - nope. Forgot to remove the input-decl